test/unit/uk/gov/hmrc/exports/config/AppConfigSpec.scala

```diff
@@ -46,6 +46,8 @@ class AppConfigSpec extends UnitSpec {
       |microservice.services.customs-data-store.host=localhost
       |microservice.services.customs-data-store.port=6790
       |microservice.services.customs-data-store.verified-email-path=/customs-data-store/eori/EORI/verified-email
+      |microservice.services.hmrc-email.host=localhost
+      |microservice.services.hmrc-email.port=8300
```

Comment:
Could you add tests for these as well? Why would this coonfig be added here?

Reply:
The tests in this spec class seem to me completely artificial. I do not understand what they are for. I tried to set a fake/wrong value in application.conf, which is where AppConfig takes the values from, and no one of the tests in this class raised any error. Can u please explain me what we are trying to verify here? Sorry, sometimes I'm a bit dumb :smiley: